test/uk/gov/hmrc/nisp/utils/BreadcrumbSpec.scala

```diff
@@ -47,6 +50,18 @@ class BreadcrumbSpec extends UnitSpec with OneAppPerSuite {
       MockBreadcrumb.generateHeaderUrl()(fakeRequestSP, nispUser) should not include "Gaps+in+your+record"
     }
 
+    "return an item text without a 'name' variable, when user.name has no value" in {
+      MockBreadcrumb.generateHeaderUrl()(fakeRequestSP, emptyNispUser) should not include ("account?name=")
```

Comment:
The name could appear in other parts of the query string. Perhaps just assert `"name="` ?